config.md

```diff
@@ -265,6 +265,41 @@ Annotations are key-value maps.
 }
 ```
 
+## VM
+
+VM is an optional object used by virtual machine-based runtimes.
+
+* **`imagePath`** (string, required) path to file that represents the root filesystem for the virtual machine.
+* **`kernel`** (object, required) specifies details of the kernel to boot the virtual machine with.
+
+Note that `imagePath` refers to a path on the host (outside of the virtual machine). This field is distinct from the **`path`** field in the (#Root-Configuration) section since in the context of a virtual machine-based runtime:
+
+* **`ImagePath`** will represent the root filesystem for the virtual machine.
+* The container root filesystem specified by **`path`** will be mounted inside the virtual machine at a location chosen by the virtual machine-based runtime.
+
+The virtual machine-based runtime will use these two path fields to arrange for **`path`** to be presented to the process to run as the root filesystem.
+
+An example of a fully-populated `VM` object:
```

Comment:
Can this example get [a header][1] for convenient linking?

[1]: https://github.com/opencontainers/runtime-spec/blob/d8d0bc6f601f98122f8908eb21158e32432ab563/style.md#examples